### ISSUE TITLE
Microsoft Edge miscalculation regarding y-axis  tickAmount

### ISSUE THREAD

**Issue by jimfilippou** [apexcharts/apexcharts.js#191]
## Codepen

https://codepen.io/jimfilippou-the-animator/pen/vQGNqz?editors=0010

## Browser

- Microsoft Edge

## Explanation

- What is the behavior you expect?
![expectation](https://user-images.githubusercontent.com/6877046/48201588-51e88980-e36b-11e8-9830-a8384abfeb1f.png)

- What is happening instead?
![reality](https://user-images.githubusercontent.com/6877046/48201604-5614a700-e36b-11e8-9a2f-140e3ee6fc3b.png)

- What error message are you getting?
No error message

**Comment by jimfilippou** [comment]
```javascript
console.log(window.navigator.userAgent);
// "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36 Edge/17.17134"
```

If that helps

**Comment by junedchhipa** [comment]
There is 1 condition for IE, which should not be returning true for Edge.
I will change the function to target only IE11.

**Comment by junedchhipa** [comment]
This commit https://github.com/apexcharts/apexcharts.js/commit/6e23336c365f6449668ad193296936d13926e012 fixes it

**Comment by jimfilippou** [comment]
I also believe this fixes the problem, when is the next build 😄 ?

**Comment by junedchhipa** [comment]
This weekend.

**Comment by junedchhipa** [comment]
Fixed in 2.2.1
